### ISSUE TITLE
Adding light image editor

### ISSUE THREAD

**Issue by micker** [FLEXIcontent/flexicontent-cck#980]
it will be interesting to add an image editor in image field
like this
https://www.imagerjs.com/

**Comment by micker** [comment]
Maybe à good idea to add à button "editing" in image field to load editor in modal
Editing original and clean thumbs
Édition l m s for specific thumbs

**Comment by micker** [comment]
this seems perfect https://scaleflex.github.io/filerobot-image-editor/

**Comment by ggppdk** [comment]
Yes we need this. Just want to check if we should use the same image editor JS as J4 or not.

**Comment by micker** [comment]
I start some test to add an editor After upload it seams a good solution
Filebot is simplier to add Just need to understand save function
